rules/linux/process_creation/proc_creation_lnx_browser_information_discovery.yaml

```
@@ -0,0 +1,37 @@
+title: Browser Information Discovery - Linux
```

```suggestion
title: Firefox Browser Information Discovery - Linux
```
As the references and filename are for firefox

---

Its better to make this rule generic by adding additional files from other browsers instead of it just being firefox focused.

@CheraghiMilad please add additional chromium browsers.